src/components/LayoutContainer/LayoutContainer.stories.tsx

```diff
@@ -8,7 +8,6 @@ export default {
   component: LayoutContainer,
   parameters: {
     axe: {
-      // TODO: re-enable when component is worked on
```

Comment:
One of the details added to the ticket for re-visiting the layout components; re-enabling the story / AXE tests